### DISCUSSION TITLE
Unable to download shuffledns package to Kali Linux

### DISCUSSION THREAD

**Discussion by heetshah27** [projectdiscovery/shuffledns#361] [Q&A]
I already installed massdns on which the shuffledns is working and I already copied the command from the github repo to install the shuffledns in my Kali Linux and after that I applied the command "apt install shuffledns" to install the shuffledns but it shows me unable to locate package again and again, now what should I do to install the shuffledns or I should use some cloud services to install the shuffledns?

**Comment by ehsandeep** [comment]
See the installation step as listed in the readme - https://github.com/projectdiscovery/shuffledns?tab=readme-ov-file#installation-instructions
